boards/arm/stm32/stm32f103-minimum/configs/adb/defconfig

```diff
@@ -40,13 +40,11 @@ CONFIG_FS_PROCFS_EXCLUDE_MOUNTS=y
 CONFIG_INIT_ENTRYPOINT="adbd_main"
 CONFIG_INIT_STACKSIZE=1024
 CONFIG_LIBUV=y
-CONFIG_LIBUV_PIPE=y
-CONFIG_LIBUV_STREAM=y
+CONFIG_LIBUV_UTILS_NONE=y
```

Comment:
remove, it's default

Reply:
It's generated by `make savedefocnfig`

Reply:
let's add default to choice in libuv/Kconfig